flang/test/Semantics/expr-errors06.f90

```diff
@@ -1,4 +1,4 @@
-! RUN: %python %S/test_errors.py %s %flang_fc1 -Werror
+! RUN: %python %S/test_errors.py %s %flang_fc1 -pedantic -Werror
```

Comment:
So this warning will not display without `-pedantic`? Is there a way to know which warnings require `-pedantic` to display?

Reply:
They are the ones enabled in flang/lib/common/Fortran-features.cpp.  Warnings can be enabled or disabled by the driver for any LanguageFeature or UsageWarning.